static/src/javascripts/projects/common/modules/user-prefs.spec.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+import userPrefs from 'projects/common/modules/user-prefs';
+import { local } from 'lib/storage';
+
+class LocalStorageMock {
+    store: Object;
+
+    constructor() {
+        this.store = {};
+    }
+
+    clear() {
+        this.store = {};
+    }
+
+    getItem(key) {
+        return this.store[key];
+    }
+
+    setItem(key, value) {
+        this.store[key] = value.toString();
+    }
+
+    removeItem(key) {
+        this.store[key] = null;
+    }
+}
+
+const localStorage = new LocalStorageMock();
+
+describe('userPrefs - Client-side preferences', () => {
+    beforeAll(() => {
```

Comment:
i don't htink these before/after are needed in jest, since it runs each test in its own process – is that correct @SiAdcock?